models/iaf_psc_alpha_hom.cpp

```diff
@@ -0,0 +1,408 @@
+/*
```

Comment:
Perhaps this could be named differently from "hom[ogeneous]", like "lookup" or "fast"? Please indicate in the model docstring how this model is different from iaf_psc_alpha.